src/cljs/rems/collapsible.cljs

```diff
@@ -56,34 +60,40 @@
   [id label-show label-hide open?]
   [:<>
    [show-more-button label-show id open? nil]
-   [show-less-button label-hide id open?]])
+   [show-less-button label-hide id open? nil]])
 
-(defn- block [id open? on-open content-always content-hideable content-footer top-less-button? bottom-less-button? class]
-  (let [always? (not-empty content-always)
-        show-more [:div.collapse-toggle
-                   [show-more-button
-                    (if always?
-                      (text :t.collapse/show-more)
-                      (text :t.collapse/show))
-                    id open? on-open]]
-        show-less [:div.collapse-toggle
-                   [show-less-button
-                    (if always?
-                      (text :t.collapse/show-less)
-                      (text :t.collapse/hide))
-                    id open?]]]
-    [:div {:class class}
-     content-always
-     (when (seq content-hideable)
-       [:div
-        (when top-less-button? show-less)
-        [:div.collapse {:id id
-                        :class (when open? "show")
-                        :tab-index "-1"}
-         content-hideable]
-        show-more
-        (when-not (false? bottom-less-button?) show-less)])
-     content-footer]))
+(defn- block [{:keys [open?]}]
+  (let [show? (r/atom open?)] ; track internal open/closed status
```

Comment:
This kind of duplicates what was previously handled by the classes/ids. Would be good to refactor this into the atom completely (with CSS animation for opening/closing). Doesn't have to happen now, but we can make sure we have an issue for it.

Reply:
yeah, currently bootstrap is handling the animation. i'll create a ticket for this

Reply:
#2961